src/ui/advanced/gestures.md

```diff
@@ -153,13 +160,40 @@ many of those widgets already respond to taps or gestures.
 For example, [`IconButton`][] and [`TextButton`][]
 respond to presses (taps), and [`ListView`][]
 responds to swipes to trigger scrolling.
-If you are not using those widgets, but you want the
+If you aren't using those widgets, but you want the
 "ink splash" effect on a tap, you can use [`InkWell`][].
 
+[`GestureDetector`]: {{site.api}}/flutter/widgets/GestureDetector-class.html
+[`IconButton`]: {{site.api}}/flutter/material/IconButton-class.html
+[`InkWell`]: {{site.api}}/flutter/material/InkWell-class.html
+[`ListView`]: {{site.api}}/flutter/widgets/ListView-class.html
+[`TextButton`]: {{site.api}}/flutter/material/TextButton-class.html
+
 ### Gesture disambiguation
 
-At a given location on screen, there might be multiple gesture
-detectors. All of these gesture detectors listen to the stream
+At a given location on screen,
+there might be multiple gesture detectors.
+For example:
+
+* A `ListTile` has a tap recognizer that responds
+  to the entire `ListTile`, and a nested one around
+  a trailing icon button. The screen rect of the
+  trailing icon is now covered by two gesture
+  recognizers that need to negotiate for who handles
+  the gesture if it turns out to be a tap.
+* A single `GestureDetector` covers a screen area
+  configured to handle multiple gestures,
+  such as a long press and a tap.
+  This gesture detector now sends multiple
```

Comment:
When reading this sentence I was wondering: WHERE to they send the recognizers TO? I really liked the wording in the previous paragraph about the recognizers negotiating. That's happening here with the long press and tap recognizer as well, maybe re-use that phrasing?

Reply:
@sfshaza2 : My LGTM depends on addressing this question. Let us know your thoughts after reviewing this.

Reply:
@atsansone, I've addressed it.